addons/l10n_in_ewaybill_stock/models/l10n_in_ewaybill.py

```diff
@@ -634,6 +636,16 @@ def _ewaybill_generate_direct_json(self):
             **self._prepare_ewaybill_tax_details_json_payload(),
         }
 
+    def _post_pdf_attachment(self, is_challan=False):
+        """ Generate and post a PDF report as a message attachment. """
+        self.ensure_one()
+        doc_label = _("Challan") if is_challan else _("E-Way Bill")
+        body = _("%s has been generated", doc_label)
+        filename = _("%s - %s.pdf") % (doc_label, self.document_number)
```

Comment:
```suggestion
        filename = "%s - %s.pdf" % (doc_label, self.document_number)
```
File name doesn't need for translation I meant to change only in case of ewaybill/challan 🙃

Reply:
Noted